NitroxClient/MonoBehaviours/Gui/MainMenu/JoinServer.cs

```diff
@@ -290,7 +303,7 @@ private void InitializeJoinMenu()
             RightSideMainMenu.groups.Add(joinServerMenu);
 
             //Not sure what is up with this menu, but we have to use the RectTransform of the Image component as the parent for our color picker panel.
-            //Most of the UI elements seem to vanish behind this Image otherwise. 
+            //Most of the UI elements seem to vanish behind this Image otherwise.
```

Comment:
I think your auto formatting settings are weird. It adds a space to every comment on the end. Can you undo these small changes?